### ISSUE TITLE
Current jQuery code isn't retrieving inputted printer values 

### ISSUE THREAD

**Issue by s1mpleman** [s1mpleman/OctoFarm#2]
Just as the title says. 

Checked all ID's and they all match, put a console log into addPrinter as that is called after addFromModal and this sent back the <input> html code. 

Check in Firefox, and it's not working in there. The current workaround is to use a variable with .value to print out the value.

**Comment by steclarkdev** [comment]
Yeah, just file all the bugs, I'll get to them. It could be the response format has changed.

**Comment by NotExpectYet** [comment]
I'll push it all to master before I go to bed then logging as I go. Cheers for this!
